hydrocarbon_problem/agents/random_agent/agent_test.py

```diff
@@ -0,0 +1,99 @@
+import chex
+
+from hydrocarbon_problem.agents.random_agent.random_agent import create_random_agent, RandomAgentState
+from hydrocarbon_problem.env.env import AspenDistillation
+from hydrocarbon_problem.env.types_ import Discount
+from hydrocarbon_problem.agents.base import SelectAction, AgentUpdate, Transition
+
+import jax
+import jax.numpy as jnp
+import numpy as np
+
+
+def create_fake_batch(env: AspenDistillation,
+                      batch_size: int = 11) -> Transition:
+    # generate fake action
+    discrete_spec, continuous_spec = env.action_spec()
+    fake_action = (discrete_spec.generate_value(), continuous_spec.generate_value())
+
+    # generate fake discount
+    discount_spec = env.discount_spec()
+    fake_discount = Discount(overall=discount_spec.overall.generate_value(),
+                             created_states=(
+                                 discount_spec.created_states[0].generate_value(),
+                                 discount_spec.created_states[1].generate_value()
+                                            )
+                             )
+
+    # generate fake next observation
+    next_observation = (env.observation_spec().generate_value(),
```

Comment:
Next observation is for both top and bottom stream, right?

Reply:
Yeah